zsh-window-title.zsh

```diff
@@ -4,139 +4,182 @@
 # Copyright © 2021 Henry Bley-Vroman
 
 'builtin' 'typeset' -g __zwt_dir && \
-	__zwt_dir=${0:A:h}
+    __zwt_dir=${0:A:h}
```

Comment:
Looks like tabs got reformatted to four spaces. Revert pls. I've added a note to CONTRIBUTING.md about how to use EditorConfig to automate some of the project's style conventions

Reply:
Sorry about that olets! ive reverted. EditorConfig was delightfully easy to set up.